resources/assets/ckeditor/plugins/filelink/plugin.js

```diff
@@ -0,0 +1,34 @@
+CKEDITOR.plugins.add( 'filelink', {
+    icons: 'filelink',
+    requires: ['iframedialog'],
+    init: function( editor ) {
+        editor.ui.addButton( 'Filelink', {
+            label: 'Insert File',
+            command: 'filelink.cmd',
+            toolbar: 'insert'
+        });
+
+        var cmd = editor.addCommand('filelink.cmd', {exec:showDialogPlugin});
+
+        CKEDITOR.dialog.addIframe(
+            'filelink.dlg',
+            'File Link',
+            editor.config.fileLinkUploadUrl,
+            800,
+            400,
+            function(){
+            }
+        );
+    }
+});
+
+function showDialogPlugin(e){
+    e.openDialog('filelink.dlg');
+}
+
+kQuery(document).ready(function($) {
+    CKEDITOR.on("instanceReady", function(e) {
+        // Remove built-in image button
+        $('#cke_21').hide();
```

Comment:
@johanjanssens We have to hide the image button instead of removing it via config i.e. `'removeButtons'  => 'Image'` as the removeButton configuration also disables the image plugin thereby disabling all images in the editor.

Reply:
@jebbdomingo Oki. No problem at all. Can you look into this in more detail? Seems a little weird we need to hide it?